### ISSUE TITLE
Epoch EXP 1.0 Issues on Bullet Buffet

### ISSUE THREAD

**Issue by orangesherbet** [EpochModTeam/Epoch#923]
All testing done on latest EXP 1.0 build as of 10/18/17 at 12:15 am CST
Testing on Bullet Buffet Server - All issues have been confirmed by myself or others

- [x] New medical items that have no effect- nanite injector, nanite cream, nanite pills, iodide pills

- [x] Metal Tower Ghost Preview needs a look (see attached screenshot)
 ![20171015230413_1](https://user-images.githubusercontent.com/4204940/31707940-ec07de04-b3aa-11e7-9896-ba366d958abe.jpg)

- [x] Metal floor cannot be built currently (double click option N/A)

- [x] Feedback on cinder / metal floors- you can craft a cinder wall but you cannot craft a cinder floor / metal floor, would be nice to be able to craft this on a workbench

- [x] Building barbed wire yields a CreateVehicle #1 Kick

- [x] Placing explosives (explosive charge, claymore charge, satchel charge) yields a CreateVehicle #0 kick

- [x] Strange CreateVehicle #1 kick when approaching towns with buildings- seems to be either a piece of furniture spawning or an antagonist causing this.  Sent AWOL my RPT and a few others' RPT files to try and figure this one out.

- [ ] Feedback on Towers- now that we have working hatches for other wall pieces- would be cool to see a working hatch for the towers!

Waiting on updates to snap building to test that further.

**Comment by morgoth0** [comment]
The crafting of cinder and metal floors, together with some other items is turned off by default in CfgEpochClient:

`	SuppressedCraftingItems[] = {	// These Items can not be crafted (but can be used in recipe)
			"VehicleRepairLg",		// Supressed for Advanced Vehicle Repair (no longer needed)
			"KitMetalFloor",		// Item is Upgradeable, but you can remove it here to make it also Craftable
			"KitMetalTower",		// Item is Upgradeable, but you can remove it here to make it also Craftable
			"KitCinderFloor",		// Item is Upgradeable, but you can remove it here to make it also Craftable
			"KitCinderTower"		// Item is Upgradeable, but you can remove it here to make it also Craftable
	};
`
Just remove the ones that you want to allow crafting. If using Advanced Vehicle Repair do **not** remove the Vehicle Repair Kit...

**Comment by orangesherbet** [comment]
oh right on- since I don't run my own server that's something I did not know!

**Comment by Ignatz-HeMan** [comment]
@morgoth0 , thanks for the hint for the supressedCraftingItems.
But I want to say one more think about that:
If you are using Advanced Vehicle Repair, it is no problem to also allow it in Crafting, but it will have no function on double click in Inventory.
We have only disabled it for crafting, because it is no longer used in Epoch (with Advanced repair enabled).
If you need it for other custom things, just enable it again.

**Comment by morgoth0** [comment]
@Ignatz-HeMan Ah... that's a fair point. I may actually enable crafting it again and require one for, for example, replacing an engine block... Thanks for the inspiration :)

**Comment by ravmustang** [comment]
Metal Floor being built handled with these two commits

https://github.com/EpochModTeam/Epoch/commit/5f6380cebfe05b82a34512e5b8e82fc28cfe434e
https://github.com/EpochModTeam/Epoch/commit/8a4954455c8893e322e60686ef55ebe74bbd9021

Crafting issue was solved by @morgoth0 's information.

Confirmed kick on building Barbed Wire.
Will pick back up after a dinner meeting for work this evening. 
Thanks @orangesherbet

**Comment by Pmba79** [comment]
The cinder tower prewiev also needs some looking into--
The opacity is turned up to high, it almost impossible to see.. 

The barbedwire fence prewiev is solid and will kill players before the fence is set.
The fence will also despawn after restart.

Cinder floors can be walked on before the prewiev runs out and the item is set in the world.

**Comment by ravmustang** [comment]
Barbed wire ghost model added here
 https://github.com/EpochModTeam/EpochCore/commit/fcd320adaacb11c1f667deaa3ec37e54f9bf31a1

Sim model added here
https://github.com/EpochModTeam/EpochCore/commit/557c8b55bdc63daeff27255ccfd65f43cc731ad5

Mission config updates here
https://github.com/EpochModTeam/Epoch/commit/9a3869bf30ebce066d29014070d41ac4afee7f46

Now that core is updated barbed wire should be corrected and ghost model displayed on build.
I will finish the cinder floor ghost aspect today and both of those tested

**Comment by ravmustang** [comment]
Cinder Floor ghost model was tested and issue corrected with Helions updated model.

Barbed Wire after another commit is working as well
https://github.com/EpochModTeam/Epoch/commit/2148634c5824d238f17d0c14436c4905d02a595d

**Comment by vbawol** [comment]
Thanks, @orangesherbet for the report and everyone for the fixes!
